Src/WitsmlExplorer.Frontend/components/Sidebar/TreeItem.tsx

```diff
@@ -32,10 +34,15 @@ const StyledTreeItem = (props: StyledTreeItemProps): React.ReactElement => {
       onIconClick={() => toggleTreeNode(props)}
       label={
         <Label>
-          {isActive && <Icon name="isActive" color={colors.interactive.primaryResting} />}
-          <NavigationDrawer selected={selected} compactMode={isCompactMode}>
-            {labelText} {isLoading && <DotProgress color={"primary"} size={32} />}
-          </NavigationDrawer>
+          {isActive && labelText == "Logs" && <Icon name="beat" color={colors.interactive.primaryResting} style={{ position: "absolute", right: "-25px", top: "6px" }} />}
```

Comment:
isActive should be sufficient to determine whether the beat icon is show. If necessary, rename `isActive` to `showBeat `and move out the logic of computing `showBeat` outside of this component